principles/3_Privacy.md

```diff
@@ -0,0 +1,51 @@
+# Privacy
+
+## Data Minimization
+
+* The minimum amount of data possible shall be collected, for the purposes for which it is gathered. Data that is not used in the disclosure to healthcare authorities should not be collected unless it is essential to the purposes of the application
+* Users can choose how much data to share from symptom surveys
+
+## Consent
+* A member of the public (MoP) shall consent to data being collected or transmitted explicitly, that is it should not be implied by usage, or mixed with another consent
+* A MoP shall provide informed consent, that is they must be able to understand what they are consenting to and the purposes for which each category of data is gathered
+* Information about privacy shall be communicated in plain English, not legal, or technical language
+* Consent to collect data shall be separate from consent to disclose data, and include retention periods
+* Consent to collect each category of data (e.g. location, proximity) shall be separate
+* No communication initiated by 3rd parties shall be provided to a MoP, through the app, without their consent
+* No data shall be shared with 3rd parties, except healthcare authorities using the agreed mechanisms
+* The user of SafePaths shall be able to disable location tracking temporarily
+* Consent to disclose data to a HA shall be separate, at the time of disclosure, and contain a reference to the HA privacy policy
+
+## Transparency & Accountability
+
+* The privacy policy shall include the steps taken by the data collector to ensure the confidentiality, integrity, and quality of the data
+* The data controller, that is, who controls what data is collected and how it is stored and disclosed shall be identified clearly
+* No single person shall be able to modify the source code and release the application to the public
+* All source code, models and technical documentation shall be publicly available, including all dependencies
```

Comment:
I'm not sure about this. That could unnecessarily restrict what libraries and infrastructure we can use? Especially on the server side. I could also see having different policies for the client side and the server side here.

Reply:
Would being explicit about what services from GCP or AWS were being used meet the "be publicly available" need?
Seems this is to assure that the MoP can evaluate what they are entrusting their data to.
Those services are public offerings, so if well identified would that serve this purpose?

Reply:
I like the current "all dependencies".  Let's say various AWS ML services started to be used, that would be very non-transparent and not in line with our mission.  If we used something not privacy impacting, e.g. S3 services, then we could risk assess that appropriately and list it as a deviation.  thoughts?

Reply:
I think it is safer to use "all dependencies" and force the addition of a deviation than to list all current specific services. I think that has a better chance of causing someone to think of what they are doing as a deviation from the standard.